### DISCUSSION TITLE
Add example / tutorial for jax.scipy.optimize.minimize

### DISCUSSION THREAD

**Discussion by rpetit** [jax-ml/jax#16036] [General] [answered]
Hi! Unless I'm mistaken, there does not seem to be any example or tutorial on how to use jax.scipy.optimize.minimize. Would it be useful that I try to write a small one? If yes, should it be added directly in the docstring as in scipy.optimize.minimize, or elsewhere?

**Comment by jakevdp** [comment, marked as answer]
Hi - thanks for the interest in this! We're not currently putting much effort into `jax.scipy.optimize` because we see [JAXOpt](https://github.com/google/jaxopt) as a better-supported successor. Our eventual goal is that `jax.scipy.optimize` would be just a lightweight wrapper of JAXOpt that provides a scipy-like API for it.

Keeping that in mind, I think we'd be happy to have a small example of optimization in the JAX docs, particularly if it makes clear that JAXOpt is the more feature-complete alternative.

**Comment by rpetit** [comment]
Hi, thanks for your reply. Sorry I was completely unaware of JAXOpt, I'll check it out first!
